website/content/v0.5/Overview/installation.md

```diff
@@ -14,7 +14,7 @@ Sidero supports several variables to configure the installation, these variables
 variables or as variables in the `clusterctl` configuration:
 
 - `SIDERO_CONTROLLER_MANAGER_HOST_NETWORK` (`false`): run `sidero-controller-manager` on host network
-- `SIDERO_CONTROLLER_MANAGER_DEPLOYMENT_STRATEGY` (`RollingUpdate`): strategy to use when updating `sidero-controller-manager`, use `Recreate` when using a single node and `SIDERO_CONTROLLER_MANAGER_HOST_NETWORK` is `true`
+`SIDERO_CONTROLLER_MANAGER_HOST_NETWORK` is `true`
```

Comment:
This remnant of the original line doesn't make sense, it should also be removed

Reply:
nice catch, thanks!